cli-plugins/insight/query-somb-reports.hbs.md

```diff
@@ -0,0 +1,140 @@
+# Query Software Bill of Materials Reports
+
+This topic tells you how to query the database to retrieve a Software Bill 
+of Materials (commonly known as SBoM) report for a particular image. The information returned is similar
+to the SBoM returned by [querying for packages and CVE's a specific image contains](query-data.hbs.md#a-idexample1aexample-1-what-packages-and-cves-does-a-specific-image-contain)
+with one major difference: SBoM reports contain Common Vulnerabilities and Exposures(commonly known as CVE) and packages for an image at a specific point in time with a specific vulnerability scan tool.
+Unlike querying for images which return all information linked to an image found in the database. 
+
+## <a id='sup-usecase'></a>Supported use cases
+
+The following are use cases supported by the CLI:
+- What SBoM reports exists for a particular image?
+- Fetching a specific SBoM report.
+
+## <a id='query-insight'></a> Query using the Tanzu Insight CLI plug-in
+
+[Install the Tanzu Insight CLI plug-in](cli-installation.hbs.md) if you have not already done so.
+
+There are two commands for querying SBoM reports.
+
+- `list` - Query for a list of existing SBoM reports associated with an image
+- `get` - Query for a specific SBoM report
+
+
+## <a id='example1'></a>Example #1: What SBoM reports exists for a specific image?
+
+Run:
+
+```console
+tanzu insight report list --digest DIGEST
+```
+
+Where:
+
+- `DIGEST` is the component version or image digest.
+
+For example:
+
+```console
+$ tanzu insight report list --digest sha256:20521f76ff3d27f436e03dc666cc97a511bbe71e8e8495f851d0f4bf57b0bab6
+1. 	UID:       	b84bd3e8-8d71-4012-a7fa-310d4406658c
+	Entity Type:  	image
+	Entity UID:  	sha256:20521f76ff3d27f436e03dc666cc97a511bbe71e8e8495f851d0f4bf57b0bab6
+	Generated At:  	2022-08-01T15:55:56Z
+	Tool:
+		Name:    trivy
+		Version: 0.45.0
+		Vendor:  anchore
+2. 	UID:       	b6bed111-24eb-4814-9cbd-bbc26dd76d7f
+	Entity Type:  	image
+	Entity UID:  	sha256:20521f76ff3d27f436e03dc666cc97a511bbe71e8e8495f851d0f4bf57b0bab6
+	Generated At:  	2022-07-15T09:01:56Z
+	Tool:
+		Name:    grype
+		Version: 0.38.0
+		Vendor:  anchore
+3. 	UID:       	510de185-3000-405e-a734-c420f64b1b94
+	Entity Type:  	image
+	Entity UID:  	sha256:20521f76ff3d27f436e03dc666cc97a511bbe71e8e8495f851d0f4bf57b0bab6
+	Generated At:  	2022-07-01T13:18:56Z
+	Tool:
+		Name:    trivy
+		Version: 0.45.0
+		Vendor:  anchore
+4. 	UID:       	a007bb9b-877c-485b-9ffc-d359586c5bfc
+	Entity Type:  	image
+	Entity UID:  	sha256:20521f76ff3d27f436e03dc666cc97a511bbe71e8e8495f851d0f4bf57b0bab6
+	Generated At:  	2022-06-01T19:28:56Z
+	Tool:
+		Name:    grype
+		Version: 0.38.0
+		Vendor:  anchore
+```
+
+## <a id='example2'></a>Example #2: What packages and CVEs did my image contain on July 17, 2022?
```

Comment:
I under that July 17 is one of the dates in the list in prior example. But for a heading, I think you can be more generic. This title confused me a bit as I was browsing the document.

I would recommend _What packages and CVEs was found in a specific scan report?_